WordOfTheDay.Api/appsettings.json

```diff
@@ -8,6 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "WordContext": "Server=(localdb)\\mssqllocaldb;Database=WordContext-03daec14-a505-48b7-8fb2-72bc85eb0fcd;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "WordContext": "Server=(localdb)\\mssqllocaldb;Database=WordContext-03daec14-a505-48b7-8fb2-72bc85eb0fcd;Trusted_Connection=True;MultipleActiveResultSets=true",
+    "RabbitMQHost": "amqp://guest:guest@localhost:5672"
```

Comment:
Can you update these connections to use Docker hosted MsSql and RabbitMq?

Reply:
Is still under development. The issue is "Microsoft.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections." I will continue to try fixing it.

Reply:
ok, create new task for this in trello please, to track this issue